kubernetes/resource_kubectl_manifest.go

```diff
@@ -635,7 +655,7 @@ func resourceKubectlManifestApply(ctx context.Context, d *schema.ResourceData, m
 		}
 		log.Printf("[INFO] %v waiting for wait conditions for %vmin", manifest, timeout.Minutes())
```

Comment:
Given that the wait_for field is now optional, please add a check verifying that at least one of field or condition block is present

Reply:
I added a check for those fields and an acceptance test to cover that case as well.